### PR TITLE
feat: allow to disable cookie consent

### PR DESCRIPTION
**Pull request by hauju**
## Motivation and Context

Explain why this change is necessary or the issue it addresses.

## Summary

Provide a short summary of changes in this pull request.

- Allow to Manage cookie consent

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

**Review by antbern — APPROVED** [review]

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -122,5 +124,3 @@ extra:
     actions:
       - accept
       - manage
-  version:
-    provider: mike
```
Comment (antbern) [inline docs/mkdocs.yml:126]: Is this no longer needed? 😨

**Review by kristifidani — APPROVED** [review]

File: docs/mkdocs.yml (hunk 1 above)
Comment (hauju) [reply docs/mkdocs.yml:126]: Was shown as duplicated key

Comment (antbern) [reply docs/mkdocs.yml:126]: Ah yes I see now that its also listed above. Great! 💯

**Review by antbern — APPROVED** [review]

**Review by antbern — APPROVED** [review]
